docs/process.md

```diff
@@ -1,3 +1,31 @@
+Development Processes
+---------------------
+
+Landing PRs
+===========
+
+ * Even after the code of a PR is approved, it should only be landed if the
+   CI on github is green, or the failures are known intermittent things.
+ * If you see an approved PR of someone without commit access (that either
```

Comment:
We could specify that approvals should be from committers. Is there even a good way to figure out who is a committer on GitHub?